tools/Google.Cloud.Docs.Snippets/CallSettingsSnippets.cs

```diff
@@ -129,5 +130,26 @@ public void Overrides()
             Topic topic = client.CreateTopic(topicName, CallSettings.FromCancellationToken(cancellationToken));
             // End sample
         }
+
+        [Fact]
+        public void RetrySettingsTiming()
+        {
+            // Sample: RetrySettingsTiming
+            // The overall operation mustn't take more than 30 seconds.
+            Expiration totalExpiration = Expiration.FromTimeout(TimeSpan.FromSeconds(30));
+
+            // The delay between one RPC finishing and when we make the next one.
+            // Each delay is double the previous one, with a maximum of 5s.
+            // The first delay is 1s, then 2s, then 4s, then 5s, then 5s, etc.
+            BackoffSettings retryBackoff = new BackoffSettings(TimeSpan.FromSeconds(1), TimeSpan.FromSeconds(5), 2.0);
```

Comment:
I was wondering about that - slightly worried about line length. Will see what happens.